tools/Unix.mk

```diff
@@ -508,7 +518,7 @@ pass2: $(NUTTXLIBS)
 # pass1 dependencies and pass1 first, then build pass2 dependencies and pass2.
 # in that case, execute 'make pass1 pass2' from the command line.
 
-$(BIN): pass1 pass2
+$(BIN): prebuild pass1 pass2
```

Comment:
but prebuild will run in parallel with pass1 and pass2.

Reply:
You're right. I totally forgot the parallel execution... let me think how I can handle that...

Reply:
Updated the PR, @xiaoxiang781216. Now we shouldn't have any problems with parallel execution...